base_tier_validation/models/tier_validation.py

```diff
@@ -170,6 +186,30 @@ def _notify_accepted_reviews_body(self):
     def validate_tier(self):
         for rec in self:
             rec._validate_tier()
+            if rec.has_comment:
```

Comment:
If logic is lengthy, better to use subroutine, and overall it will look nicer, i.e.,
```
rec._validate_tier()
rec._add_comment()
rec._update_counter()
```